apps/ledger-live-desktop/src/renderer/families/hedera/types.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+
+import type {
+  Account,
+  Transaction,
+  TransactionStatus,
+  FeeStrategy,
+} from "@ledgerhq/live-common/lib/types";
+
+export type SendAmountProps = {
+  account: Account,
+  transaction: Transaction,
+  status: TransactionStatus,
+  onChange: Transaction => void,
+  updateTransaction: (updater: any) => void,
+  mapStrategies?: FeeStrategy => FeeStrategy & { [string]: * },
```

Comment:
Very minor, but these 2 props seem unused, and this is causing a failure in that CI test: https://github.com/LedgerHQ/ledger-live/runs/7099372918?check_suite_focus=true